README.md

```diff
@@ -253,6 +253,21 @@ Install `atuin` shell plugin in zsh, bash, or fish with [Fig](https://fig.io) in
 
 <a href="https://fig.io/plugins/other/atuin" target="_blank"><img src="https://fig.io/badges/install-with-fig.svg" /></a>
 
+## Uninstall
+  
+To remove Atuin you can use your distribution package manager. Example for Ubuntu 20.04:
+
+```
+sudo apt remove atuin
+```
+Then you have to comment out these two lines from your `~/.bashrc` file:
+  
+```
+[[ -f ~/.bash-preexec.sh ]] && source ~/.bash-preexec.sh
+eval "$(atuin init bash)"
+```
```

Comment:
This assumes bash. I think we can have the user refer above for their shell to see which lines to remove.

We should also not really recommend users remove the dependencies like preexec incase they need it for something else that might break if removed

Reply:
I've did some few changes to address it, please check.